misc/discoverynode/src/udmi/core.py

```diff
@@ -19,10 +21,10 @@
 class UDMI:
```

Comment:
seems like this should be `class core:` to match the filename -- or filename to match? udmi_core?  Just `UDMI` seems a bit problematic because at some point it might conflict with the actual UDMI package/class...

Reply:
Thoughts on this?